.circleci/php-5.5_web.txt

```diff
@@ -1,15 +0,0 @@
-test_metrics
```

Comment:
For consistency with all other versions, I removed this and consolidated all web tests in the same job. This approach was very valid, though, so I started this effort https://github.com/DataDog/dd-trace-php/tree/labbati/separate-web-tests/.circleci that has to be completed. The long term is to run consistently tests (quickly) in CI and to define them only in one place. See branch